src/fparser/two/utils.py

```diff
@@ -714,9 +714,15 @@ def match(
                     # We've found the enclosing end statement so break out
                     found_end = True
                     break
+
+                # Deal with any following comments, includes, and/or directives
+                DynamicImport.add_comments_includes_directives(content, reader)
+
                 if not strict_order:
                     # Return to start of classes list now that we've matched.
                     i = 0
+                elif once_only:
```

Comment:
Suggest a comment: we had a match for this sub-class and `once_only` is set so move on to the next sub-class.

Reply:
Added comment.